Rex.podspec

```diff
@@ -0,0 +1,136 @@
+#
```

Comment:
Generally, we should strip out comments like this from Podspecs. Anything that starts with `#` basically